tests/Behat/OAuthContext/OAuthContext.php

```
@@ -243,49 +260,101 @@ public function unauthorizedErrorShouldBeReturned(): void
+    {
+        $existingClient = $this->entityManager->getRepository(Client::class)->find($id);
+        if ($existingClient) {
+            $this->entityManager->remove($existingClient);
+            $this->entityManager->flush();
+            $this->entityManager->clear();
+            $this->verifyClientRemoval($id);
+        }
     }
 
-    private function approveAuthorization(): void
+    private function verifyClientRemoval(string $id): void
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (Naming Rules)

(ShortVariable)